tests/headers_test.cc

```diff
@@ -238,6 +239,18 @@ TEST(headers_test, date_test_asctime) {
     ASSERT_EQ(dd3, expected_time_point);
 }
 
+TEST(headers_test, date_test_ostream) {
+
+    using namespace std::chrono;
+
+    std::ostringstream os;
```

Comment:
I think that "std::chrono: namespace is unused here?

Reply:
I've removed this namespace in next commit.